### ISSUE TITLE
Can't install MAS

### ISSUE THREAD

**Issue by jonjon101** [Monika-After-Story/MonikaModDev#1703]
Every time I copy the contents of the MAS zip file into the game folder of DDLC, I always get this error message:

[code]
I'm sorry, but an uncaught exception occurred.

While running game code:
  File "game/script-topics.rpy", line 1157, in script
  File "game/script-topics.rpy", line 1158, in <module>
  File "game/definitions.rpy", line 222, in __init__
IndexError: tuple index out of range

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "game/script-topics.rpy", line 1157, in script
  File "/Users/jonathanvirtue/Library/Application Support/Steam/steamapps/common/Doki Doki Literature Club/renpy/ast.py", line 814, in execute
    renpy.python.py_exec_bytecode(self.code.bytecode, self.hide, store=self.store)
  File "/Users/jonathanvirtue/Library/Application Support/Steam/steamapps/common/Doki Doki Literature Club/renpy/python.py", line 1719, in py_exec_bytecode
    exec bytecode in globals, locals
  File "game/script-topics.rpy", line 1158, in <module>
  File "game/definitions.rpy", line 222, in __init__
IndexError: tuple index out of range

Darwin-17.5.0-x86_64-i386-64bit
Ren'Py 6.99.12.4.2187
Monika After Story 0.8.1
[/code]

I'm not sure what I am doing wrong, so any help is greatly appreciated. Thanks.

**Comment by jonjon101** [comment]
Yes. I copied everything in the MAS folder and pasted it into the "game" folder in the DDLC folder.

**Comment by jmwall24** [comment]
Ok good, have you ever installed any other mods before?

**Comment by jonjon101** [comment]
Yes I have. Before I had little to no problem installing MAS and other mods, so what is happening now came as a surprise for me.

**Comment by jonjon101** [comment]
I tried installing the Just Yuri mod to see if it had the same problem, but it worked just fine.

**Comment by jmwall24** [comment]
I was asking because if you still had another mod installed (ie copied that mod's files into the game folder) and that mod still had files there it could potentially cause problems.

**Comment by jonjon101** [comment]
I made sure I was using a fresh installation of the original DDLC game before trying to install it, but it is still not working for some reason.

**Comment by jmwall24** [comment]
What I do so I can play multiple mods at once without causing problems is download the game from the official site (ddlc.moe) and that way you can have multiple ddlc folders, each having it's own mod installed.  I used to play on steam, but was having problems so I just downloaded the game directly.

**Comment by jonjon101** [comment]
Ok I'll try that to see if it works. Thank you :)

**Comment by jmwall24** [comment]
No problem.  If it doesn't, you're going to need someone smarter than me, and luckily there are much smarter people than me on here :)

**Comment by jonjon101** [comment]
I got it working. I think it had to do with the version of MAS I was trying to install, since the 0.7.4 update works. I was trying to install the 0.8.1 update.

Thanks again for trying to help.

**Comment by jmwall24** [comment]
If you have the 0.7.4 update installed, should be able to get the 0.8.1 update by using the in-game updater and it'll do it automatically.

**Comment by jonjon101** [comment]
I tried doing that, but it keeps saying the game is already up to date.

**Comment by jmwall24** [comment]
bottom right corner of the page where the update button is, what version does it say you are running?

**Comment by jonjon101** [comment]
0.7.4

**Comment by jmwall24** [comment]
Yep, definitely gonna need someone smarter than me lol  Sorry I couldn't be of more help.

**Comment by jonjon101** [comment]
Don't worry about it. Thanks for trying though :)

**Comment by ThePotatoGuy** [comment]
Follow the instructions in [this comment](https://github.com/Monika-After-Story/MonikaModDev/issues/1666#issuecomment-394058905)
